diff --git a/Makefile b/Makefile
index 762985d..4a052df 100644
--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@ tar: clean
 
 .PHONY: install
 install:
-	install -d $(PREFIX)/{bin,etc,lib,share/oorb}
+	install -d $(PREFIX)/{bin,etc,lib,share/openorb}
 
 	for P in $(PROGRAMS); do \
 		install bin/$$P $(PREFIX)/bin; \
@@ -91,7 +91,7 @@ install:
 	install -m644 lib/liboorb.a         $(PREFIX)/lib
 	install       lib/liboorb.$(LIBEXT) $(PREFIX)/lib
 	install -m644 main/oorb.conf        $(PREFIX)/etc
-	cp -a $(shell find data -maxdepth 1 -type f) $(PREFIX)/share/oorb
+	cp -a $(shell find data -maxdepth 1 -type f) $(PREFIX)/share/openorb
 
 ifeq ("$(PYOORB)","1")
 	install       python/$(shell cat python/pyoorb.name) $(shell echo $${PYTHON_SITE_PATH:-$$(cat python/pyoorb.sp_dir)})
diff --git a/modules/planetary_data.f90 b/modules/planetary_data.f90
index 364cc93..b4822f2 100644
--- a/modules/planetary_data.f90
+++ b/modules/planetary_data.f90
@@ -1581,7 +1581,7 @@ CONTAINS
        END IF
     END DO
 
-    OORB_DATA_DIR = resolveDirectory("share/oorb", "OORB_DATA")
+    OORB_DATA_DIR = resolveDirectory("share/openorb", "OORB_DATA")
 
     IF (LEN_TRIM(OORB_DATA_DIR) == 0) THEN
        OORB_DATA_DIR = "."
diff --git a/main/oorb.f90 b/main/oorb.f90
index a766cb5..7e7d92e 100644
--- a/main/oorb.f90
+++ b/main/oorb.f90
@@ -522,7 +522,7 @@ PROGRAM oorb
   END IF
 
   ! Set path to Gnuplot scripts using the environment variable if it exists:
-  gnuplot_scripts_dir = resolveDirectory("share/oorb/gnuplot", "OORB_GNUPLOT_SCRIPTS_DIR")
+  gnuplot_scripts_dir = resolveDirectory("share/openorb/gnuplot", "OORB_GNUPLOT_SCRIPTS_DIR")
 
   ! Read observation file if given:
   obs_fname = get_cl_option("--obs-in="," ")
