--- build/Makefile
+++ build/Makefile
@@ -59,9 +59,7 @@ $(addprefix ../bin/, $(PROGRAMS)): ../bin/%: ../main/%
 	cp -a $< $@
 
 $(addprefix ../main/, $(PROGRAMS)): ../main/%: %.o ../lib/liboorb.a
-	$(FC) $(FCOPTIONS) $^ $(ADDLIBS) -o $@.tmp
-	fix-rpath.sh $@.tmp
-	mv $@.tmp $@
+	$(FC) $(FCOPTIONS) $^ $(ADDLIBS) -o $@
 
 #
 # Build version include file. This is included by oorb.f90 to
@@ -117,10 +115,10 @@ pyoorb: ../lib/$(PYOORB_SO)
 		  exit -1; \
 		}
 	@ [[ ! -z "$(LDFLAGS)" ]] && echo "$$LDFLAGS" | grep -E -v -q -- "(-bundle|-shared)" && { \
-	  [[ `uname` == "Darwin" ]] && LDFLAGS="$$LDFLAGS -bundle" || LDFLAGS="$$LDFLAGS -shared"; \
+	  [[ `uname` == "Darwin" ]] && LDFLAGS="$$LDFLAGS" || LDFLAGS="$$LDFLAGS"; \
 	}; \
-	F77= $(F2PY) --quiet -m pyoorb pyoorb.o pyoorb.pyf ../lib/liboorb.a --build-dir ./_pyoorb_build -c --noarch \
-	        $(F2PY_FCOMPILER) --f90exec=$(FC) --f90flags="$(FC_INC)../build $(FCOPTIONS)" $(ADDLIBS) && \
+	F77= $(F2PY) --verbose ${CONDA_LIBORB} --build-dir ./_pyoorb_build -c --noarch \
+	        $(F2PY_FCOMPILER) --f90exec=$(FC) --f90flags="$(FC_INC)../build $(FCOPTIONS) ${FFLAGS}" $(ADDLIBS) && \
 	mv $(PYOORB_SO) ../python
 	@ echo
 	@ # FIXME: this exists so `sudo make install` knows what to install. It's hacky this way
